apps/02RockPaperScissors.js

```diff
@@ -6,11 +6,48 @@ prompt.start();
 
 
 function rockPaperScissors(hand1, hand2) {
+    var computerChoice = Math.random();
+    var hand1 = hand1.toLowerCase();
```

Comment:
remove the var in front of hand1 since here you're only overwriting the variable.